src/boot/main.c

```diff
@@ -346,7 +346,11 @@ void thread3_main(UNUSED void *arg) {
         gIsConsole = FALSE;
         gBorderHeight = BORDER_HEIGHT_EMULATOR;
         gIsVC = IS_VC();
```

Comment:
Can this or the one at the top be removed? Not sure why this is getting called twice.

Reply:
Got rid of unnecessary gIsVC. if I rememeber right, it was done early to allow using gIsVC in EEPROM setup so the one inside `IO_READ(DPC_CLOCK_REG) == 0` must be unnecessary.